src/Sulu/Bundle/DocumentManagerBundle/DependencyInjection/SuluDocumentManagerExtension.php

```diff
@@ -101,6 +101,7 @@ public function prepend(ContainerBuilder $container)
                     'exception' => [
                         'codes' => [
                             'Sulu\Component\DocumentManager\Exception\DocumentNotFoundException' => 404,
+                            'Sulu\Component\DocumentManager\Exception\VersionNotFoundException' => 404,
```

Comment:
This PR is already targeted at master, so it will be a hotfix. But I would like to wait with the release, in case other stuff comes up.